README.md

```diff
@@ -26,6 +26,7 @@ Certified by the Mattermost team, and tested prior to each Mattermost server rel
 
 #### Unofficial
 Non-production plugins built by Mattermost staff and community.
+- [Standup Raven](https://github.com/standup-raven/standup-raven)
```

Comment:
This list should be in alphabetical order. Would you please move this line at the end?

Reply:
Done. Didn't realize that. Thanks!